### ISSUE TITLE
Update HSBC

### ISSUE THREAD

**Issue by jimmyvanau** [jdavis/twofactorauth#6301]
### Site name

HSBC

### Site URL

https://hsbc.com

### Update reason

Other (please describe below).

### Additional information

tl;dr Suggest changing the current `hsbc.com` to `hsbc.co.uk` as that is the linked documentation (with region uk).

----
HSBC has each region implement internet banking and 2FA in its own way. While similar, each region can do their own thing - each region often has its own app. Generally, it's strongly encouraged (you're directed to login with a second factor), however, it is possible to bypass and login with reduced functionality. For example, for HSBC US, I believe MFA is not fully supported -> https://www.us.hsbc.com/online-banking/security-device/device-faq/ has the FAQ

> "There is no way I am going to keep another fob on my key ring or carry this around. Isn't there a way that I can just do my online banking like I used to without using this? 
> You do still have the option to log on to Personal Internet Banking without your HSBC Security Device, but you will not be able to conduct certain transactions."

Additionally, from the domain hsbc.com, if one heads to Online Banking in the navigation bar -> https://www.hsbc.com/#online-banking then country selection is mandated, then users are redirected to any of us.hsbc.com, hsbc.co.uk, hsbc.com.ar, etc

I would therefore like to suggest that current `hsbc.com` entry is updated to `hsbc.co.uk`. Happy to submit a PR if this sounds OK 

Can also submit second PR for HSBC US not (fully) supporting MFA. This will be a duplicate of #4393 which seems to have disappeared. #3872 and #4253 are also relevant.

### Issue Eligibility

- [X] The issue I'm creating is not a duplicate of an existing [issue](https://github.com/2factorauth/twofactorauth/issues).
- [X] The issue I'm creating is not a duplicate of an existing [pull request](https://github.com/2factorauth/twofactorauth/pulls)

**Comment by phallobst** [comment]
Issue has been resolved by PRs #6302 and #6303.
